### PR TITLE
Deployment plan for CEX

### PR DESCRIPTION
**Pull request by taokayan**
Resolves https://github.com/eosnetworkfoundation/eos-evm/issues/586
relocated to Deployment plan for CEX

### CODE REVIEW

File: deployments_plan_for_CEX/readme.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+...
+  "earliest_available_block_num": 302853021,
+  "last_irreversible_block_time": "2023-06-23T03:10:35.500"
+}
+```
+- in the above example all EVM blocks before `"last_irreversible_block_time": "2023-06-23T03:10:35.500"` are irreversible. Use the time conversion script:
+`
+python3 -c 'from datetime import datetime; print(hex(int((datetime.strptime("2023-06-23T03:10:35.500","%Y-%m-%dT%H:%M:%S.%f")-datetime(1970,1,1)).total_seconds())))'
+`
+to get the EVM irreversible blocktime in hex `0x64950d2b`. By scanning every EVM block, we found out that the EVM blocks up to ```6828746``` are irreversible, because its timestamp is `0x64950d2b`:
```
Comment (arhag) [inline deployments_plan_for_CEX/readme.md:216]: Should we use the mixhash and relate to the EOS block ID to confirm the correct block was found?

**Review by arhag — COMMENTED** [review]
LGTM

File: deployments_plan_for_CEX/readme.md (hunk 1 above)
Comment (taokayan) [reply deployments_plan_for_CEX/readme.md]: added the mixHash check. please review again.

**Comment by nsjames** [comment]
This lgtm. 

To make things easier in the future we should merge in the excellent changes from the pinax team for the miner: https://github.com/pinax-network/eos-evm-miner/

Some of the key improvements are:
- Better errors
- Better logging & metrics
- Routing all requests to the node, so GETs also flow through the miner

**Review by arhag — APPROVED** [review]

**Pull request merged**